src/Avalonia.Controls.DataGrid/DataGrid.cs

```diff
@@ -716,6 +716,16 @@ public DataGridRowDetailsVisibilityMode RowDetailsVisibilityMode
             set { SetValue(RowDetailsVisibilityModeProperty, value); }
         }
 
+        public static readonly RoutedEvent<RoutedEventArgs> DisplayRegionChangedEvent = RoutedEvent.Register<DataGrid, RoutedEventArgs>(
```

Comment:
You also should be able to use DataGrid.LayoutUpdated event.

Reply:
Thanks very much. The LayoutUpdated event works fine with my app